web/src/lib/cms/previews/TaskPreview.tsx

```diff
@@ -21,9 +22,20 @@ const TaskPreview = (props: PreviewProps): ReactElement => {
     }),
   });
 
+  const displayContent = createEmptyTaskDisplayContent();
+
+  const roadmap = generateRoadmap({});
+
+  const CMS_ONLY_disable_overlay = true;
   return (
     <div className="cms" ref={ref} style={{ margin: 40, pointerEvents: "none" }}>
-      <TaskElement task={task} CMS_ONLY_fakeBusiness={fakeBusinessWithMunicipality} />
+      {renderTaskPageComponent({
```

Comment:
[dust] Is this something that needs to be using function-call syntax and can't be a React component?

Reply:
done